noark-extraction-validator/src/main/java/com/documaster/validator/config/delegates/StorageConfiguration.java

```diff
@@ -32,6 +32,10 @@
 	@Parameter(names = DATABASE_NAME, description = "The name of the database")
 	private String databaseName = "xdb";
 
+	private static final String DATABASE_DIR_LOCATION = "-db-dir-location";
+	@Parameter(names = DATABASE_DIR_LOCATION, description = "The location of the dir where the database will be stored")
+	private String databaseDirLocation = System.getProperty("java.io.tmpdir") + "/noark-extraction-validator-db";
```

Comment:
I would actually prefer to leave it as a string since if I use `Files.createTempDir()` I will create an empty temporary dir each execution, even if the storage type is not hsqldb_file and that dir will not really be needed/used. Also I actually need the path as String really, not an as object.
I don't worry so much about the `java.io.tmpdir` because `Files.createTempDir()` uses it internally: `private static final Path tmpdir = Paths.get((String)AccessController.doPrivileged(new GetPropertyAction("java.io.tmpdir")));`

Though to be honest when I started writing the implementation I went first for this exact approach.